content/en/real_user_monitoring/browser/setup/server.md

```diff
@@ -0,0 +1,135 @@
+---
+title: RUM Browser Monitoring Server Side Instrumentation
+beta: true
+code_lang: server
+type: multi-code-lang
+code_lang_weight: 2
+further_reading:
+- link: '/real_user_monitoring/explorer/'
+  tag: 'Documentation'
+  text: 'Learn about the RUM Explorer'
+- link: '/logs/log_collection/javascript/'
+  tag: 'Documentation'
+  text: 'Learn about the Datadog Browser SDK for Logs'
+---
+
+{{< callout url="https://www.datadoghq.com/private-beta/rum-sdk-auto-injection/" btn_hidden="false" header="Access the Preview!">}}
+RUM SDK Injection in Preview. Sign up for access! You can also request early access if you are using Apache HTTPD, Tomcat, or another web server technology.
+{{< /callout >}}
+
+## Overview
+
+Datadog RUM Server Side Instrumentation (SDK Injection) lets you opt into Real User Monitoring (RUM) automatically by instrumenting web applications served through a web server or proxy.
+
+RUM Server Side Instrumentation works by injecting a RUM SDK JavaScript scriptlet into the HTML responses being served through a web server or proxy.
+
+After your applications have been instrumented, you can begin configuring your RUM application in Datadog.
+
+## Prerequisites
+
+The automatic installation method requires that you have the [Datadog Agent][2] installed.
+
+## Setup your RUM application
+
+<div class="alert alert-warning">To request support for a web server that is not listed here, <a href="https://www.datadoghq.com/private-beta/rum-sdk-auto-injection/">fill out this form.</a></div>
+
+{{< tabs >}}
+{{% tab "Nginx" %}}
```

Comment:
I would style it as NGINX everywhere, since that's how NGINX itself does it, but I'm not sure if you're using existing RUM styles here. Just wanted to flag!

Reply:
Yes, I was torn on this. Seems like it's either NGINX or nginx, but to follow our existing styles, I decided to go with Nginx.